src/arrayMethodSort.js

```diff
@@ -5,7 +5,26 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 1; i < this.length; i++) {
+      for (let j = 0; j < this.length - 1; j++) {
```

Comment:
What's the purpose of two nested loops? Actually, one loop is enough

Reply:
Its default buble sort algorithm , without second loop - result will be incorect

Reply:
you can use [`do while` loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/do...while), if you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review